In [1]:
import pandas as pd
import numpy as np
from scipy import stats

###  Business Problem is given as 
Is the male-female buyer rations are similar across regions

## Data description is given as 

α is given as 0.05 (95% Confidence)

Y is a Discrete data

X is a Discrete data 

Chi Square test is performed as there are two variables

## Chi-Square Test

In [2]:
df = pd.read_csv("BuyerRatio.csv")

H0 = The male-female buyer rations are similar across regions  
H1 = The male-female buyer rations are not similar across regions

In [3]:
stats.chi2_contingency([df["East"], df["West"], df["North"], df["South"]])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value of Chi-Square test  == 0.66 > α

## Thus HO is accepted.  
## The male-female buyer rations are similar across regions  